lib/brainstem/concerns/controller_dsl.rb

```diff
@@ -233,6 +238,41 @@ def title(text, options = { nodoc: false })
           configuration[brainstem_params_context][:title] = \
             options.merge(info: text)
         end
+
+
```

Comment:
Intentional extra newline? Perhaps we should think about bringing in our Rubcop lint rules to this repo.